src/Bonami/Collection/TrySafe.php

```diff
@@ -216,6 +221,15 @@ public function recoverWith(callable $callable): TrySafe
                 })->flatMap($id);
             }
 
+            public function recoverWithPF(callable $case, callable $recovery): TrySafe
```

Comment:
Somehow im not sure about introducing concept of pf into collections (do we mention it elsewhere?)
What about just `recoverWithIf` w/ the exact same semantics?

Reply:
Renamed